README.md

```diff
@@ -94,6 +94,8 @@ The owner ID is similar to the command role, however this is the ID of the user
 
 The prefix is the command prefixed used for all the commands in the bot on the Discord side, by default this is set it `!`.
 
+The messageMode can either be `bot` or `webhook`. This selects how the messages should be displayed when sent from Minecraft to Discord. To see an example of what webhook mode looks like [click here](https://i.imgur.com/tulcMVA.png) and for bot mode [click here](https://i.imgur.com/07EPheLh.jpg).
```

Comment:
Another few notes would be
1.  It would be a good idea to include that the bot will need the `Manage Webhooks` permission, either for the entire server or for the channel the bot is using to send messages in for the webhook option to actually work, without it the bot won't be able to create or find the webhooks used to send messages with.
2. The Discord rate limit for webhooks is 30 requests every 60 seconds, while the normal bot-message rate limit is 5 messages every 5 seconds, so using webhooks effectively halves the number of messages the bot would be able to send per minute, during high activity in a guild this might cause some issues.